CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [16.2.1] - TBD
+### Added
+* A `DEVELOPERS.md` file with some notes on the inner workings of the project. 
```

Comment:
This isn't really part of the release so I don't think you need to update the CHANGELOG.